### DISCUSSION TITLE
Power Drop-outs?

### DISCUSSION THREAD

**Discussion by Lunchtime0614** [doudar/SmartSpin2k#396] [Q&A]
Anyone get power drop outs randomly?  99% of the time I have no issues, no drops at all.  Other times I get a ride like this where halfway through something goes crazy and it starts dropping the power meter....then after a few min it seems to be fine again.  I wouldn't even have said anything about this other than it's happened on the last two rides.  It's normally many rides in between when I see it do this.  Of course I have no logs since I was in the middle of a prescribed ride.

I don't know if it's some ghost signal from something else in my house  - kid walking by with bluetooth headphones or something or what...nothing's changed in my setup.

![Screen Shot 2022-11-02 at 4 02 27 PM](https://user-images.githubusercontent.com/68966864/199591740-3fa199ba-a2cd-4672-863f-0e1778adcbcf.png)

**Comment by eMadman** [comment]
Looks like it's self-healing.  What happens if you press on the up/down shifter buttons together for 3 seconds?  That forces a rescan of bluetooth and may get you going again.  I haven't experienced this with my power meter.  I'm curious if it's the DFC dropping signal

How long are these drops?

Also, out of curiosity, are you still experiencing wifi issues?

**Reply by Lunchtime0614** [reply]
I don't have the shifters connected since I'm just in ERG mode 100% of the time

I'm not using the DFC for this - I have assioma pedals - so I don't know if it's dropping the assioma signal or the signal from SS2K to the app (computer in this case)

Dropout length of time ranges from a few seconds to 20-40 seconds.


-No wifi issues since I separated the 2.4 and 5ghz into separate SSID's.

**Reply by Lunchtime0614** [reply]
I've thought about removing/rescanning - before my next ride - sometimes with other devices - like headphones you have to do that periodically.

**Reply by eMadman** [reply]
OK, cool - no wifi issues is great!  That's an indicator that bluetooth on the ss2k should be working since wifi/bluetooth run off the same radio.  

Shifters might be useful to have connected just to force the reset mid-ride.  It'll bring the 20-40 seconds down by a fair bit and won't interrupt your ride.
